domains/bearshen.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Bearshenmin",
+    "email": "[email protected]"
```

you can safely remove the email if you dont want to share
```suggestion
    "username": "Bearshenmin"
```